### PR TITLE
OCM-3599: added image_overrides to version type

### PR DESCRIPTION
**Pull request by cristianoveiga**

### CODE REVIEW

File: model/clusters_mgmt/v1/billing_model_item_type.model [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// BillingModelItem represents a billing model
+struct BillingModelItem {
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/billing_model_item_type.model:18]: I wonder if this should be class? class have ID and HREF and Kind by default.

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: model/clusters_mgmt/v1/billing_model_item_type.model (hunk 1 above)
Comment (cristianoveiga) [reply model/clusters_mgmt/v1/billing_model_item_type.model:18]: Yes, it makes sense ... I copied it over from the accounts_mgmt folder, but I believe we better use the class in this case. 

I will updated it in both sides. cc: @tirthct

Comment (cristianoveiga) [reply model/clusters_mgmt/v1/billing_model_item_type.model]: @tzvatot I've replaced the struct it with a class. PTAL

**Review by renan-campos — APPROVED** [review]
/lgtm

File: model/accounts_mgmt/v1/billing_model_item_type.model [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// BillingModelItem represents a billing model
+class BillingModelItem {
+	// BillingModelType is the type of the BillingModel. e.g. standard, marketplace
```
Comment (zgalor) [inline model/accounts_mgmt/v1/billing_model_item_type.model:19]: missing period at the end of each remark

**Review by zgalor — APPROVED** [review]
/lgtm

**Pull request merged**